src/bridge/RoomAccessSyncer.ts

```diff
@@ -265,6 +291,26 @@ export class RoomAccessSyncer {
         await botClient.sendStateEvent(roomId, "m.room.power_levels", plContent, "");
     }
 
+    public async onFailedMessage(req: BridgeRequest, server: IrcServer, channel: string) {
+        const key = `${server.getNetworkId()}:${channel}`;
+        const currThreshold = this.accessRefreshCount.get(key) || 0;
```

Comment:
```suggestion
        const currThreshold = (this.accessRefreshCount.get(key) || 0) + 1;
```
And then remove the +1s in the following lines for better readability.
I found it confusing why the current count (which I assumed included the current failure) needed to be one higher.